File: Examples/Features/Charts/Axes/ConfigTickMarkLabel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Charts.Axes
{
    public class ConfigTickMarkLabel : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];

            GrapeCity.Documents.Excel.Drawing.IShape shape = worksheet.Shapes.AddChart(GrapeCity.Documents.Excel.Drawing.ChartType.ColumnClustered, 250, 20, 360, 230);
            worksheet.Range["A1:D6"].Value = new object[,]
            {
                {null, "S1", "S2", "S3"},
                {1, -25, 25, 25},
                {2, 51, 36, 27},
                {3, 52, 80, 30},
                {4, 22, -20, 65},
                {5, 23, 69, 69}
            };
            shape.Chart.SeriesCollection.Add(worksheet.Range["A1:D6"], GrapeCity.Documents.Excel.Drawing.RowCol.Columns, true, true);
            GrapeCity.Documents.Excel.Drawing.IAxis category_axis = shape.Chart.Axes.Item(GrapeCity.Documents.Excel.Drawing.AxisType.Category);
            GrapeCity.Documents.Excel.Drawing.IAxis value_axis = shape.Chart.Axes.Item(GrapeCity.Documents.Excel.Drawing.AxisType.Value);

            //config tick label's format
            category_axis.TickLabelPosition = GrapeCity.Documents.Excel.Drawing.TickLabelPosition.NextToAxis;
            category_axis.TickLabelSpacing = 2;
            category_axis.TickLabels.Font.Color.RGB = Color.DarkOrange;           
            category_axis.TickLabels.Font.Size = 12;
            category_axis.TickLabels.NumberFormat = "#,##0.00";
            value_axis.TickLabels.NumberFormat = "#,##0;[Red]#,##0";
        }
    }
}

File: Examples.VB/Features/Charts/Axes/ConfigTickMarkLabel.vb
Imports System.Drawing

Namespace Features.Charts.Axes
    Public Class ConfigTickMarkLabel
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)

            Dim shape As IShape = worksheet.Shapes.AddChart(ChartType.ColumnClustered, 250, 20, 360, 230)

            worksheet.Range("A1:D6").Value = New Object(,)  {
                {Nothing, "S1", "S2", "S3"},
                {1, -25, 25, 25},
                {2, 51, 36, 27},
                {3, 52, 80, 30},
                {4, 22, -20, 65},
                {5, 23, 69, 69}
            }

            shape.Chart.SeriesCollection.Add(worksheet.Range("A1:D6"), RowCol.Columns, True, True)

            Dim category_axis As IAxis = shape.Chart.Axes.Item(AxisType.Category)

            Dim value_axis As IAxis = shape.Chart.Axes.Item(AxisType.Value)

            'config tick label's format
            category_axis.TickLabelPosition = TickLabelPosition.NextToAxis
            category_axis.TickLabelSpacing = 2
            category_axis.TickLabels.Font.Color.RGB = Color.DarkOrange
            category_axis.TickLabels.Font.Size = 12
            category_axis.TickLabels.NumberFormat = "#,##0.00"
            value_axis.TickLabels.NumberFormat = "#,##0;[Red]#,##0"
        End Sub
    End Class
End Namespace
